yarn.lock

```diff
@@ -108,13 +108,27 @@
     "@endo/promise-kit" "^0.2.56"
     node-fetch "^2.6.0"
 
-"@agoric/cosmic-proto@0.2.2-dev-a5437cf.0", "@agoric/cosmic-proto@0.3.1-dev-ff36f02.0+ff36f02", "@agoric/cosmic-proto@^0.3.0":
+"@agoric/cosmic-proto@0.2.2-dev-a5437cf.0":
   version "0.2.2-dev-a5437cf.0"
   resolved "https://registry.yarnpkg.com/@agoric/cosmic-proto/-/cosmic-proto-0.2.2-dev-a5437cf.0.tgz#cba81f0455ba1875d3f389b69d93e36d5569bfb3"
   integrity sha512-ocwgjLUJcuKeDP7/RHWQXDqpDDCEEBqsX1JQNjmnkljE5dNkUBiDNMXurzmD+SZJqGDZ1HxaQFfF/9zo99q4vA==
   dependencies:
     protobufjs "^7.0.0"
 
+"@agoric/cosmic-proto@0.3.1-dev-ff36f02.0+ff36f02":
+  version "0.3.1-dev-ff36f02.0"
+  resolved "https://registry.yarnpkg.com/@agoric/cosmic-proto/-/cosmic-proto-0.3.1-dev-ff36f02.0.tgz#f6d6cf21b62e4fffbbc3c5634d3754d17d793f70"
+  integrity sha512-y266YxgI6ZN8UxqDezFjfcHTTOlo9FhFM22cN9EQVah0Z4FbNzXplYwrNRZZljGvtbb33w+oZNfv89fZ2aB5Gw==
+  dependencies:
+    protobufjs "^7.0.0"
+
+"@agoric/cosmic-proto@^0.3.0":
+  version "0.3.0"
```

Comment:
May be worth opening a ticket to get `@agoric/cosmic-proto` down to a single version. Seeing 3 here

Reply:
Hmm yea, created https://github.com/Agoric/wallet-app/issues/137